### PR TITLE
 onnx scale layer and tensorflow fc layer bias 

### PR DESCRIPTION
**Pull request by JiahaoYao**
onnx scale layer and tensorflow fc layer bias

### CODE REVIEW

**Comment by zhantong** [comment]
Hi Jiahao, thank you for your contribute! But [`Scale`](https://github.com/onnx/onnx/blob/master/docs/Operators.md#Scale) is still an experimental Operator in ONNX, and also the test environment ONNX-TensorFlow does not support `Scale`, so the correctness of `emit_scale()` is unable to be determined.

I'd suggest that we add the support for `Scale` when it became a **non-experimental** operator.

**Comment by JiahaoYao** [comment]
@zhantong , the `Scale` op is defined in [Caffe Scale Layer](http://caffe.berkeleyvision.org/tutorial/layers/scale.html). I think it might be different from the `Scale` in onnx. 
Therefore, The `Emit_Scale` is for this scale layer. The implement relies on `batchnorm`, just setting mean to zero and var to one.

**Comment by zhantong** [comment]
Wow~ sorry I didn't noice it uses `BatchNormalization` rather than `Scale`, my bad. Great work!

**Comment by JiahaoYao** [comment]
@zhantong Thank you! 
But I think it remains the problem because in the future when [`Scale`](https://github.com/onnx/onnx/blob/master/docs/Operators.md#Scale) checks in, `Emit_Scale` might be in conflict.
